<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>888</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>1400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/window_icon/C:/Users/Miguel/Desktop/Images/Icon.ico</normaloff>:/window_icon/C:/Users/Miguel/Desktop/Images/Icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(240, 248, 255);
background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/top_bar/top_bar2.png);
background-position:center</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="content">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="center">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>700</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>700</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 41, 53);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_logo">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>90</y>
            <width>360</width>
            <height>90</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/logo/logo_360x90.png);
background-repeat:no-repeat;
background-position:center</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="frame_pfp">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>185</y>
            <width>120</width>
            <height>120</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame {	
	background-image: url(:/profile-pic/avatar.png);
	border-radius:60px;
	border:3px solid rgb(255,207,0);
	background-position:center
}
QFrame:hover {
	transform: scale(1.2)
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_login">
          <property name="geometry">
           <rect>
            <x>175</x>
            <y>320</y>
            <width>250</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: rgb(45, 45, 45);
	border-radius:5px;
	padding:5px;
	border:2px solid rgb(55,55,55);
}
QLineEdit:hover {
	background-color: rgb(65, 65, 65);
}
QLineEdit:focus {
	color: rgb(200,200,200);
	padding:5px;
	background-color: rgb(65, 65, 65);
	border:2px solid rgb(255,207,0);
	
}</string>
          </property>
          <property name="maxLength">
           <number>32</number>
          </property>
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_pwd">
          <property name="geometry">
           <rect>
            <x>175</x>
            <y>375</y>
            <width>250</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: rgb(45, 45, 45);
	border-radius:5px;
	padding:5px;
	border:2px solid rgb(55,55,55);
}
QLineEdit:hover {
	background-color: rgb(65, 65, 65);

}
QLineEdit:focus {
	color: rgb(200,200,200);
	padding:5px;
	background-color: rgb(65, 65, 65);
	border:2px solid rgb(255,207,0);
	
}</string>
          </property>
          <property name="maxLength">
           <number>32</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_remember">
          <property name="geometry">
           <rect>
            <x>175</x>
            <y>430</y>
            <width>120</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
	border: 1px solid rgb(255, 207, 0);
	width:15px;
	height: 15px;
	border-radius: 8px;
	background-color: rgb(65, 65, 65);
}
QCheckBox::indicator::checked {
	border: 1px solid rgb(255, 207, 0);
	background-color: rgb(209, 164, 1);
}</string>
          </property>
          <property name="text">
           <string>Remember me</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_enter">
          <property name="geometry">
           <rect>
            <x>175</x>
            <y>500</y>
            <width>250</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(45, 45, 45);
	border:2px solid rgb(55,55,55);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(65, 65, 65);
	border: 2px solid rgb(255, 207, 0);
}
QPushButton:pressed {
	color: rgb(45, 45, 45);
	background-color: rgb(255, 207, 0);
	border: 2px solid rgb(255, 207, 0);
}</string>
          </property>
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_error">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>600</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 0);
border-radius: 5px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="label_error">
             <property name="font">
              <font>
               <family>OCR A Extended</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Error</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_close_error">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	border-radius:5px;
	background-position:center;
	background-image: url(:/error-icon/cil-x.png);
	background-color: rgb(55, 0, 0);
}
QPushButton:hover {
	background-color: rgb(50, 50, 50);
	color:rgb(200,200,200)
}
QPushButton:pressed {
	background-color: rgb(35, 35, 35);
	color:rgb(200,200,200)
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <zorder>frame_error</zorder>
         <zorder>frame_logo</zorder>
         <zorder>frame_pfp</zorder>
         <zorder>lineEdit_login</zorder>
         <zorder>lineEdit_pwd</zorder>
         <zorder>checkBox_remember</zorder>
         <zorder>pushButton_enter</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottom_bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/bottom_bar/background_bottom.png);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_credits">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 207, 0);</string>
         </property>
         <property name="text">
          <string>Created by Miguel Victor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
